src/averageAges.js

```diff
@@ -37,7 +38,20 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const getAverageWomen = (!withChildren)
```

Comment:
Please rename, only name of functions should start with verbs. But here we have an array.
```suggestion
  const filteredWomen = (!withChildren)
```